### PR TITLE
Make level 5 in autopark project.

### PR DESCRIPTION
**Pull request by aces47**
Add Rent class.
Add VehicleComrarer class that implements an Icomprarer<Vehicle> interface.
Add Collections class.
Add types.csv, vehicles.csv, rent.csv files which store information.

### CODE REVIEW

File: AutoParkConsole/Classes/Rent.cs [hunk 1]
```diff
@@ -6,25 +6,20 @@
...
-		public DateTime GetRentDate() => _RentDate;
-		public void SetRentCost(decimal rentCost) => _RentCost = rentCost;
-		public decimal GetRentCost() => _RentCost;
+		public DateTime RentDate { get; set; }
+		public decimal RentCost { get; set; }
 
 		public Rent()
 		{
 
 		}
```
Comment (PWMaestro) [inline AutoParkConsole/Classes/Rent.cs:17]: Why are you left empty string between brackets?

Comment (aces47) [reply AutoParkConsole/Classes/Rent.cs]: > Why did you leave empty string between brackets?

Fixed this problem in all files. Thak you.

**Review by PWMaestro — APPROVED** [review]

File: AutoParkConsole/Classes/Collections.cs [hunk 2]
```diff
@@ -5,18 +5,15 @@
 
 namespace AutoParkConsole.Classes
 {
-	class Collections
+	internal class Collections
```
Comment (PushMeNow) [inline AutoParkConsole/Classes/Collections.cs:8]: You shouldn't use any libs for csv parsing. Write please yourself parsing and remove TextFieldParser

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: AutoParkConsole/Classes/Collections.cs [hunk 3]
```diff
@@ -76,60 +72,35 @@ List<Vehicle> LoadVehicle(string inFile)
 
 			return result;
 		}
-		Vehicle CreateVehicle(string[] rows)
+		Vehicle CreateVehicle(string[] csvData)
```
[the remark this answers is not in the document]
Comment (aces47) [reply AutoParkConsole/Classes/Collections.cs]: Fixed

File: AutoParkConsole/Classes/Collections.cs (hunk 2 above)
Comment (aces47) [reply AutoParkConsole/Classes/Collections.cs:7]: Fixed

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**